src/css/layout/ScrollbarLayout.css

```diff
@@ -27,23 +27,18 @@
 .ScrollbarLayout/mainHorizontal {
   height: var(--scrollbar-size);
   left: 0;
-  transition-property: background-color height;
-}
-
-/* Touching the scroll-track directly makes the scroll-track bolder */
-.ScrollbarLayout/mainHorizontal.public/Scrollbar/mainActive,
-.ScrollbarLayout/mainHorizontal:hover {
-  height: var(--scrollbar-size-large);
```

Comment:
Removed `scrollbar-size-large` since it was changing the height of the scrollbar wrapper element thus in turn very slightly modifying the layout of the table.